app/layout/scenarios/edit/features/add/list/component.tsx

```diff
@@ -80,27 +80,29 @@ export const ScenariosFeaturesAddList: React.FC<ScenariosFeaturesAddListProps> =
             </div>
           )}
 
-          {allFeaturesData && allFeaturesData.map((item, i) => {
-            const selectedIndex = selected.findIndex((f) => f === item.id);
-
-            return (
-              <div
-                key={`${item.id}`}
-                className={cx({
-                  'border-t border-dashed': i !== 0,
-                })}
-              >
-                <Item
-                  {...item}
-                  scrollRoot={scrollRef}
-                  selected={selectedIndex !== -1}
-                  onToggleSelected={() => {
-                    handleToggleSelected(item.id);
-                  }}
-                />
-              </div>
-            );
-          })}
+          {allFeaturesData && allFeaturesData
+            .sort((feature) => (feature.isCustom ? -1 : 0)) // Display custom features first
+            .map((item, i) => {
+              const selectedIndex = selected.findIndex((f) => f === item.id);
+
+              return (
+                <div
+                  key={`${item.id}`}
+                  className={cx({
+                    'border-t border-dashed': i !== 0,
+                  })}
```

Comment:
[minor / disregard if gibberish]
(not a CSS expert, but maybe it can get some custom, named thing/selector like `:not(:first-child)` ?

Reply:
Had to look it up, but you're right. Even though it looks like there's not a `:not` in tailwind, the effect can be replicated fully with Tailwind.